config/settings.yml

```diff
@@ -3,9 +3,51 @@
   :ems_hawkular:
     :event_handling:
       :event_groups:
+
+:ems_refresh:
+  :hawkular:
+    :refresh_interval: 15.minutes
+
+:event_handling:
+  :event_groups:
+    :application:
+      :critical:
+      - hawkular_datasource.error
+      - hawkular_datasource_remove.error
+      - hawkular_deployment.error
+      - hawkular_deployment_remove.error
+      - hawkular_event.critical
+      :detail:
+      - hawkular_datasource.ok
+      - hawkular_datasource_remove.ok
+      - hawkular_deployment.ok
+      - hawkular_deployment_remove.ok
+      - hawkular_event
+
 :http_proxy:
   :hawkular:
     :host:
     :password:
     :port:
     :user:
+
+:workers:
+  :worker_base:
+    :defaults:
+      :event_catcher:
+        :defaults:
```

Comment:
shouldn't these lines 36 and 38 be deleted? The original structure in the config [was](https://github.com/ManageIQ/manageiq/blob/82637f6960f3737f01ae990eb3288b44187b6abd/config/settings.yml#L1158:L1197):

```yaml
  :workers:
    :worker_base:
      :event_catcher:
        :event_catcher_hawkular:
```

Or am I missing something?